publish/2019-10-20-WhatsNew020.adoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "What's New in Release 0.2.0"
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/u/50876789
+author_github: https://github.com/stephenkinder
+seo-title: What is new in release 0.2.0
+seo-description: What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides.
+blog_description: "What is new in the 0.2.0 release - this covers the new namespace support, ease of use enhancements and guides."
```

Comment:
blog_description: What is new in 0.2.0 - New CLI, Collections enhanced, Kubernetes Application Navigator, Guides, Kabanero operator enhanced